test/screeningsList.test.js

```diff
@@ -0,0 +1,54 @@
+import { getScreeningsList } from '../server/screeningsList.js';
+import { mockApiAdapter } from './mockApiAdapter.js'
+import { jest, test, describe, expect } from '@jest/globals';
+
+
+describe('Control if data recieved has correct format and size', () => {
+
+    beforeEach(() => {
+        jest.useFakeTimers();
+        jest.setSystemTime(new Date(2023, 2, 2));
+    });
+
+    afterEach(() => {
+        jest.clearAllTimers();
+    });
+    test('Filters out screenings that are older than the current date', async () => {
```

Comment:
All four tests use the exact same input (the `mockApiAdapter`) meaning that they will all have the exact same output. So although there are four tests, they are actually only testing one case. Consider using more versatile testing data, and use different data to test different scenarios.

Reply:
Thanks for your feedback!